packages/components/README.md

```diff
@@ -32,7 +32,7 @@ _renderGrid options_
 | fetchGifs                               | `(offset:number) => Promise<GifsResult>` | undefined | A function that returns a Promise<GifsResult>. Use `@giphy/js-fetch-api`                              |
 | columns                                 | `number`                                 | 3         | The number of columns in the grid                                                                     |
 | gutter                                  | `number`                                 | 6         | The space between columns and rows                                                                    |
-| noResultsMessage                        | `string                                  |           | element`                                                                                              | undefined | Customise the "No results" message |
+| noResultsMessage                        | `string`                                 | undefined | Customise the "No results" message |
```

Comment:
@fvasco guessing the `|` in the `string | element` broke the formatting, can you replace the `|` with the word `or`? That's what we have for the React package's README